### PR TITLE
Github Action to build and upload linux wheels

### PR DESCRIPTION
**Pull request by michelp**

### CODE REVIEW

File: .github/workflows/wheels.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      matrix:
+        os: ["ubuntu-latest"]
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Build manylinux Python wheels
+      uses: RalfG/python-wheels-manylinux-build@v0.3.4-manylinux2014_x86_64
+      with:
+        python-versions: 'cp37-cp37m cp38-cp38 cp39-cp39'
+        build-requirements: 'cffi numpy cython'
```
Comment (eriknw) [inline .github/workflows/wheels.yml:24]: Can we be more explicit with the NumPy version?  We should build using NumPy 1.18.* as of today according to [NEP 29](https://numpy.org/neps/nep-0029-deprecation_policy.html).

File: suitesparse.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    VERSION=${BASH_REMATCH[1]}
+else
+    exit -1
+fi
+
+curl -L https://github.com/DrTimothyAldenDavis/GraphBLAS/archive/refs/tags/v${VERSION}.tar.gz | tar xzf -
+cd GraphBLAS-${VERSION}/build
+cmake ..
+make -j$(nproc)
+make install
```
Comment (eriknw) [inline suitesparse.sh:13]: Small nits:
- should we be explicit and specify `-DCMAKE_BUILD_TYPE=Release`?  IIRC, it used to default to Debug, but I think it now defaults to Release.
- any value in building with more verbosity, `VERBOSE=1`?

**Review by eriknw — COMMENTED** [review]
Hooray :tada:

Oh, and can we also automate uploading source distribution to PyPI?  You know, the result of `python setup.py sdist`.  This allows people to install SuiteSparse:GraphBLAS through other means.

File: .github/workflows/wheels.yml (hunk 1 above)
Comment (michelp) [reply .github/workflows/wheels.yml:24]: Done

File: suitesparse.sh (hunk 2 above)
Comment (michelp) [reply suitesparse.sh:13]: > Small nits:
> 
>     * should we be explicit and specify `-DCMAKE_BUILD_TYPE=Release`?  IIRC, it used to default to Debug, but I think it now defaults to Release.

Why not!  Added.  I'm no cmake expert but it does look to default to Debug.
> 
>     * any value in building with more verbosity, `VERBOSE=1`?

On cmake?  Not sure what this produces, if it's useful to you I'll add it.

**Comment by michelp** [comment]
> Hooray tada
> 
> Oh, and can we also automate uploading source distribution to PyPI? You know, the result of `python setup.py sdist`. This allows people to install SuiteSparse:GraphBLAS through other means.

Yep, I'll address this in a followup PR

**Pull request merged**